timesync/peersync/sync.go

```diff
@@ -132,10 +132,10 @@ func New(h host.Host, peers getPeers, opts ...Option) *Sync {
 
 // Sync manages background worker that compares peers time with system time.
 type Sync struct {
-	errCnt uint32
+	errCnt atomic.Uint32
```

Comment:
Why does this need to be an `atomic.Uint32`? It seems to only be modified within the same method and goroutine, so there shouldn't be any concerns about races?

Reply:
You are absolutely right. TBH I just mindlessly modified it from using `atomic.LoadUint32` and such to this, without thinking if this really needs to be an atomic. In fact, it doesn't even need to be a field on the `Sync` struct, but a local variable.